packages/blade/rollup.config.js

```diff
@@ -66,7 +66,7 @@ const getWebConfig = ({ exportCategory }) => ({
       sourcemap: true,
     },
   ],
-  external: [/@babel\/runtime/],
+  external: (id) => id.includes('@babel/runtime'),
```

Comment:
the regexes are just haunting so better made it simple. this will mark all the `@babel/runtime*` deps external